docs/js-guide/code-organization.rst

```diff
@@ -15,15 +16,15 @@ which we structure as follows:
        font/
        images/
        js/       <= JavaScript
-       less/
+       scss/
        lib/      <= Third-party code: This should be rare, since most third-party code should be coming from yarn
        spec/     <= JavaScript tests
        ...       <= May contain other directories for data files, i.e., `json/`
      templates/myapp/
        mytemplate.html
 
-\* There are a few places we do intentionally use script blocks, such as
```

Comment:
The less.js script block isn't relevant because we haven't allowed that in-browser compiling method in a long time. I just deleted the whole block because I couldn't think of another example?

Reply:
They're rare, but there are a few: third party analytics stuff in register new user is one place, some printing code in reports. (Found those examples from running `./scripts/codechecks/hqDefine.sh list-script`.)

Reply:
updating